packages/shared/src/error.ts

```diff
@@ -190,6 +190,7 @@ const DefaultMessages = Object.freeze({
   InvalidProxyUrlErrorMessage: `The proxyUrl passed to Clerk is invalid. The expected value for proxyUrl is an absolute URL or a relative path with a leading '/'. (key={{url}})`,
   InvalidPublishableKeyErrorMessage: `The publishableKey passed to Clerk is invalid. You can get your Publishable key at https://dashboard.clerk.com/last-active?path=api-keys. (key={{key}})`,
   MissingPublishableKeyErrorMessage: `Missing publishableKey. You can get your key at https://dashboard.clerk.com/last-active?path=api-keys.`,
+  MissingClerkProvider: `{{source}} must be wrapped within the <ClerkProvider> component. Please see: https://clerk.com/docs/quickstarts/react#configure-clerk-provider`,
```

Comment:
The quickstart might change in the future so this is brittle to link to. We should link to the reference page. If that lacks details we need for this error to be useful, we should PR to the Clerk docs :)

Also removed the "see" to cater for people who have vision impairment / who are blind. (minor thing)

```suggestion
  MissingClerkProvider: `{{source}} must be wrapped within the <ClerkProvider /> component. Learn more: https://clerk.com/docs/components/clerk-provider`,
```